tests/testthat/test-add_multi_tracks.R

```diff
@@ -0,0 +1,76 @@
+test_that("add_multi_tracks() returns list", {
```

Comment:
Right, makes sense. I spent very less time on the tests and did something there to have some tests so that bioc doesnt error out.
Feel free to improve or replace them :)